src/d_player.h

```diff
@@ -181,6 +181,8 @@ typedef struct player_s
   // [Woof!] show centered "A secret is revealed!" message
   char*               secretmessage;
 
+  int                 btuse, btuse_tics;
```

Comment:
If we make these values part of `struct player_t`, are we going to save them in savegames?

Reply:
If I'm not mistaken, the struct members are only saved if we explicitly add code that writes them into savegames. I don't think these are worth saving.